outputs.tf

```
@@ -2,3 +2,13 @@ output "workspace_id" {
   description = "The ID of this Databricks workspace."
   value       = azurerm_databricks_workspace.this.id
 }
+
+output "workspace_name" {
+  description = "The name of this Databricks workspace."
+  value       = azurerm_databricks_workspace.this.name
+}
+
+output "workspace_url" {
+  description = "The url of this Databricks workspace."
```

Capitalize abbreviation.

```suggestion
  description = "The URL of this Databricks workspace."
```

---

> It seems like the values for public_subnet_name and private_subnet_name might've been mixed up though?

Will take a look

> Also, is there a specific reason why you've added variables for public_subnet_id and private_subnet_id, then parse the name from those variables and pass the to the corresponding *_name arguments? Instead of just getting the subnet names from the variables instead? Same question for the virtual_network_id variable.

Just a preference on using id instead of name as a variable, and to decrease the needed input variables by one. But its not a super hard preference and it could maybe be cleaner to do it like the resource.